encrypted-media-respec.html

```diff
@@ -1664,12 +1664,7 @@
                             <li><p>Clear all stored session data associated with this object, including the <a def-id="sessionId"></a> and license destruction record.</p>
                               <p class="note">A subsequent call to <a def-id="load"></a> with the value of this object's <a def-id="sessionId"></a> would fail because there is no data stored for that session ID.</p>
                             </li>
-                            <li>
-                              <p>
-                                Run the <a def-id="update-key-statuses-algorithm"></a> algorithm on the <var>session</var>, providing an empty sequence.
```

Comment:
Good change.  We should make this regardless of which PR, possibly as an independent change along with some of the other cleanup.